README.md

```diff
@@ -100,7 +116,16 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+  Our database may have many individual tables, and each table will have a schema
+  which refers to the attributes it has (and what data types they are), relationships
+  with other tables (i.e. foreign key of another table), as well as any constraints
+   it may contain. It is like a blueprint or a way to describe the individual tables,
+    where as an ERD is like a blueprint for all the tables and how they interact with
+     each over as a whole. In the people and wishes example, we would have two separate
+     database tables (people and wishes). A person may have the attributes (or columns)
+      name (text), age (int), and number_of_wishes (int). The wishes table might have
+       a foreign key which points to the person that owns the wish, and a wished_at_date
+        (date) which would specify when a person wished this particular wish.
 ```
```

Comment:
"Blueprint" is correct, but a schema actually models all of the tables as well. The schema maps your ERD onto the database by describing its structure in the language of the database -- in our case, SQL.